### DISCUSSION TITLE
Provenance Blockchain Community Call

### DISCUSSION THREAD

**Discussion by iramiller** [provenance-io/provenance#557] [Announcements]
**Scheduled:** Nov 22, 2021 at 9:00 AM to 10:00 AM, MST (repeats every other week)

This is the biweekly Provenance Blockchain Community of users call hosted by the Figure Blockchain Protocol Engineering team. We meet every other Monday to review the technical road map and get feedback from the community.

This is a great place to connect if you are an engineer using the Provenance Blockchain to build an application, or are an engineer contributing to the Provenance blockchain itself.  We welcome new users, veterans, and engineers just wanting to check things out and learn about the community.

We start each call with a brief review of the current development efforts including new issues, pull requests, and upcoming changes.  We finish with an open discussion/QA from the attendees.

**Agenda:** https://hackmd.io/Dh8y3WlrTQCo0BVO1gAd5Q
-:::::::::::::::::::::::::::::::::::::::::::::-
Do not edit this section of the description.
This event has a video call.
Join: https://meet.google.com/bpf-phac-hbg
(US) +1 725-696-2797 PIN: 226007190#
View more joining options: https://tel.meet/bpf-phac-hbg?pin=4778442920597&hs=7
-:::::::::::::::::::::::::::::::::::::::::::::-
[calendar-invite.ics](https://github.com/provenance-io/provenance/files/7566374/invite.ics.txt)

**Comment by blazeapps007** [comment]
I checked the code , what new stuffs does it adds besides what all VM's created using the cosmos SDK normally serves . is the chain even needed ? why not use cosmos itslef ?
